### DISCUSSION TITLE
引入react 博客中样式挂掉了

### DISCUSSION THREAD

**Discussion by lkzwc** [walinejs/waline#1931] [Q&A]
![image](https://user-images.githubusercontent.com/84896877/231412758-555bd68f-5317-44d6-9ad9-9e4cbd47e498.png)
![image](https://user-images.githubusercontent.com/84896877/231413820-b90117d1-c1d5-4297-a4e7-c1d92d64e904.png)

**Comment by Mister-Hope** [comment]
Are you sure you import css into your project?

**Reply by lkzwc** [reply]
import "@waline/client/waline-meta.css"  引入了 感觉好像没生效 直接在首页引入就可以了吧

**Comment by lkzwc** [comment]
![image](https://user-images.githubusercontent.com/84896877/231466347-a6f0c00b-9ed0-4c69-ac25-b43f75aed823.png)

**Comment by lkzwc** [comment]
dd
